chapters/6-analysis.tex

```diff
@@ -745,6 +745,29 @@ \subsection{Using {\LaTeX} for dynamic research outputs}
 so papers can be more easily be formatted into their specific layout.
 
 \section{Looking ahead}
+This section discussed the steps needed to take original data
+and create analysis datasets and outputs from it.
+This process created new derived information
+from measurements taken in the field or collected by partners,
+and combined these measures with the research design
+described in the data map and the analysis plan.
+Doing so is difficult, creative work,
+and it is impossible to be reproduced by another person
+without access to detailed records and explanations
+of how you interpreted and modified the data.
+Creating organized and well-documented code
+for both the creation of new information
+and the creation of research outputs describing
+patterns you identified in the data --
+and used to answer your research questions --
+provides this knowledge to both future users of the original data
+as well as readers interested in completely understanding
+the results you obtained.
```

Comment:
Overly complex sentence. perhaps:
You learned that code must be well-organized and well-documented to allow others to understand how research outputs were created and how those were used to answer the research questions. 
?

Reply:
```suggestion
You learned that code must be well-organized and well-documented 
to allow others to understand how research outputs were created 
and how those were used to answer the research questions.
```